Shopify/cli
# [Feature] Add schema and type ref to flow action

### WHY are these changes introduced?

- [Ticket](https://github.com/Shopify/flow/issues/19269)

### WHAT is this pull request doing?

- Adds return type ref and schema as optional fields to the Flow action extension

### How to test your changes?

- Create a Flow action extension in spin, check the resulting extension config in partners

### Post-release steps



### Checklist
